### PR TITLE
Align AppSearchViewController with emission search button

### PR DESCRIPTION
**Pull request by sarahscott**
This aligns the search bar on Home in eigen with the search bar in emission

transition

### CODE REVIEW

File: Artsy/View_Controllers/App_Navigation/ARTopMenuNavigationDataSource.m [hunk 1]
```diff
@@ -52,6 +52,10 @@ - (instancetype)init
     }
 
     ARHomeComponentViewController *homeVC = [[ARHomeComponentViewController alloc] init];
+
+    // Necessary to ensure UIKit doesn't automagically throw off our top margins
+    homeVC.automaticallyAdjustsScrollViewInsets = NO;
```
Comment (sarahscott) [inline Artsy/View_Controllers/App_Navigation/ARTopMenuNavigationDataSource.m]: RN can change [`automaticallyAdjustContentInsets`](https://facebook.github.io/react-native/docs/scrollview.html#automaticallyadjustcontentinsets) but not `automaticallyAdjustsScrollViewInsets`

Comment (alloy) [inline Artsy/View_Controllers/App_Navigation/ARTopMenuNavigationDataSource.m]: Hmm, you’ll have to show me what the difference is tomorrow :)

**Comment by sarahscott** [comment]
This should green.

**Comment by sarahscott** [comment]
This also does NOT change the status bar animation or color; those will come as separate PRs.

**Comment by alloy** [comment]
Not going to wait for CI, which is probably going to timeout.

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:warning:</td>
      <td data-sticky="true">Detected some Swift building time outliers</td>
    </tr>
  </tbody>
</table>

Time | Class | Function |
| --- | ----- | ----- |241.3ms | [AuctionViewController.swift](/artsy/eigen/tree/sarah-status-bar-styling/Artsy/View_Controllers/Auction/AuctionViewController.swift#L199) | @objc dynamic func showRefineTappedAnimated(animated: Bool)
200.6ms | [SwiftyJSON.swift](/artsy/eigen/tree/sarah-status-bar-styling/Pods/SwiftyJSON/Source/SwiftyJSON.swift#L445) | get {}
164.8ms | [WebSocket.swift](/artsy/eigen/tree/sarah-status-bar-styling/Pods/Starscream/Source/WebSocket.swift#L528) | private func processRawMessage(buffer: UnsafePointer<UInt8>, bufferLen: Int)
104.5ms | [LiveAuctionLotSetViewController.swift](/artsy/eigen/tree/sarah-status-bar-styling/Artsy/View_Controllers/Live_Auctions/LiveAuctionLotSetViewController.swift#L29) | init(salesPerson: LiveAuctionsSalesPersonType, traitCollection: UITraitCollection)
100.9ms | [LiveAuctionLotSetViewController.swift](/artsy/eigen/tree/sarah-status-bar-styling/Artsy/View_Controllers/Live_Auctions/LiveAuctionLotSetViewController.swift#L185) | @objc func setupToolbar()
99.9ms | [LiveAuctionLotViewController.swift](/artsy/eigen/tree/sarah-status-bar-styling/Artsy/View_Controllers/Live_Auctions/LiveAuctionLotViewController.swift#L47) | @objc override func viewDidLoad()
73.5ms | [AuctionLotMetadataStackScrollView.swift](/artsy/eigen/tree/sarah-status-bar-styling/Artsy/View_Controllers/Live_Auctions/Views/AuctionLotMetadataStackScrollView.swift#L18) | required init(viewModel: LiveAuctionLotViewModelType, salesPerson: LiveAuctionsSalesPersonType, sideMargin: String)
72.7ms | [LiveAuctionsAdminViewController.swift](/artsy/eigen/tree/sarah-status-bar-styling/Artsy/View_Controllers/Live_Auctions/LiveAuctionsAdminViewController.swift#L22) | @objc override func viewDidLoad()
72.6ms | [Observable.swift](/artsy/eigen/tree/sarah-status-bar-styling/Pods/Interstellar/Sources/Observable.swift#L71) | private final func nextTokenHash() -> Int
66.7ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/sarah-status-bar-styling/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L25) | func ar_presentBlurredOverlayWithTitle(title: String, subtitle: String, buttonState: BlurredStatusOverlayViewCloseButtonState = default)
64.1ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/sarah-status-bar-styling/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L31) | (closure)
62.9ms | [AuctionViewController.swift](/artsy/eigen/tree/sarah-status-bar-styling/Artsy/View_Controllers/Auction/AuctionViewController.swift#L291) | @objc dynamic func embeddedModelsViewController(controller: AREmbeddedModelsViewController!, didTapItemAtIndex index: UInt)
59.7ms | [LiveAuctionToolbarView.swift](/artsy/eigen/tree/sarah-status-bar-styling/Artsy/View_Controllers/Live_Auctions/Views/LiveAuctionToolbarView.swift#L48) | func setupUsingState(lotState: LotState)
58.3ms | [LiveAuctionFancyLotCollectionViewLayout.swift](/artsy/eigen/tree/sarah-status-bar-styling/Artsy/View_Controllers/Live_Auctions/Views/LiveAuctionFancyLotCollectionViewLayout.swift#L156) | func interpolateFrom(a: CGFloat, to b: CGFloat, ratio: CGFloat) -> CGFloat


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Pull request merged**